Edition-01/Starwars-Demo/src/StarWarsData.ps1

```diff
@@ -106,23 +108,23 @@ function Get-SWPerson {
 
     if ($null -ne $person.Error)
```

Comment:
```suggestion
    if ($null -eq $person)
```
Looks like a valid request for a nonexistent person returns an empty array with your new API, which `Invoke-StarWarsApi` returns as null.

Reply:
Thanks for spotting that! This has been fixed.